### PR TITLE
[statistics] Undefined index errors in Imaging Statistics

### PR DESCRIPTION
**Pull request by intralizee**
### Brief summary of changes

Inside the Imaging of the statistics module, undefined index errors were populated and I used isset to catch them.

### This resolves issue...

- [x] Redmine? #
https://redmine.cbrain.mcgill.ca/issues/15256

- [ ] Github? #

### To test this change...

- [ ] Go to Loris->statistics->Imaging

### CODE REVIEW

File: tools/single_use/Normalize_Consent_Data.php [hunk 1]
```diff
@@ -99,8 +99,8 @@
         $consentID  = $entry['ID'];
         $candID     = $entry['CandID'];
         // Check if consent status is not_answered
-        if($status === "not_answered") {
-            array_push($errors, "Deprecated consent status 'not_answered' found for " . $consentName . ":
+        if($status === "not_answered" || $status === "") {
+            array_push($errors, "Deprecated consent status 'not_answered' or empty string found for " . $consentName . ":
```
Comment (driusan) [inline tools/single_use/Normalize_Consent_Data.php]: I think these merit different error messages. `not_answered` is deprecated, empty string was never really valid in the schema, but MySQL's stupid handling of enums sometimes inserts it anyways.

File: tools/single_use/Normalize_Consent_Data.php [hunk 2]
```diff
@@ -122,8 +122,8 @@
         if($status === "no" && !empty($date)) {
             if(empty($withdrawal)) {
                 array_push($errors, "The date of withdrawal is missing for " . $consentName . ":
-                           [ID]     => " . $consentID . "
-                           [CandID] => " . $candID . "
+                           [ID]     => " . $consentID . "		
+                           [CandID] => " . $candID . "
```
Comment (driusan) [inline tools/single_use/Normalize_Consent_Data.php]: I'm a little surprised it passes Travis with the extra whitespace at the end of the line added.

**Comment by johnsaigle** [comment]
@driusan Yes I meant on the `Imaging Statistics` tab. The notices were generated there.

File: modules/statistics/php/stats_mri.class.inc [hunk 3]
```diff
@@ -113,20 +113,32 @@ class Stats_MRI extends \NDB_Form
...
+                    $tpl_data['data'][$subproj]['total'] += $row['val'];
+                } else {
+                    $tpl_data['data'][$subproj]['total'] = $row['val'];
+                }
+                $tpl_data['data'][$vl][$subcat] += $row['val'];
+                $tpl_data['data'][$vl]['total'] += $row['val'];
                 $tpl_data['data'][$subproj][$C][$vl][$subcat]  = $row['val'];
                 $tpl_data['data'][$subproj][$C][$vl]['total'] += $row['val'];
                 $tpl_data['data'][$subproj][$C][$subcat]      += $row['val'];
                 $tpl_data['data'][$subproj][$C]['total']      += $row['val'];
```
Comment (johnsaigle) [inline modules/statistics/php/stats_mri.class.inc]: The notices I posted earlier are still appearing. Please don't dismiss my review manually. I'll check back on this PR and test it again when changes are made.

**Comment by johnsaigle** [comment]
I've taken this over in #4129

**Pull request closed without merging**